src/App/Fossa/Test.hs

```diff
@@ -68,11 +68,9 @@ testMain config = runStickyLogger SevInfo $
             TestOutputJson -> logStdout . decodeUtf8 . Aeson.encode $ issues
             TestOutputPretty -> pure ()
         n -> do
-          logError $ "Test failed. Number of issues found: " <> pretty n
           if null (issuesIssues issues)
-            then logError "Check the webapp for more details, or use a full-access API key (currently using a push-only API key)"
+            then logError "Cannot display issues in the terminal. Check the webapp for more details, or use a full-access API key (currently using a push-only API key)."
```

Comment:
[nit] wording: consider something like "A push-only API key was used, so we can't display issue details. Check the webapp for issue details, or run this command again with a full-access API key"